runtime/doc/starting.txt

```diff
@@ -470,14 +468,11 @@ accordingly.  Vim proceeds in this order:
 	This does the same as the command: >
 		:runtime! plugin/**/*.vim
 		:runtime! plugin/**/*.lua
-<	The result is that all directories in the 'runtimepath' option will be
-	searched for the "plugin" sub-directory and all files ending in ".vim"
-	and ".lua" will be sourced (in alphabetical order per directory),
-	also in subdirectories. First all the "*.vim" files will be sourced and
-	then all the "*.lua" files will be sourced. If two files with same
-	name but different extensions exists they will be treated in same
-	manner. For example when both "foo.vim" and "foo.lua" exists then
-	first "foo.vim" will be sourced then "foo.lua" will be sourced.
+<	The result is that all directories in 'runtimepath' will be searched
+	for the "plugin" sub-directory and all files ending in ".vim" or
+	".lua" will be sourced (in alphabetical order per directory),
+	also in subdirectories. First "*.vim" are sourced, then "*.lua" files.
```

Comment:
docs should really avoid unclear, redundant excursions. There are 2 sentences here that don't add any value.